### DISCUSSION TITLE
Python: - "Please compile first." on Run command

### DISCUSSION THREAD

**Discussion by romankurnovskii** [cpeditor/cpeditor#1176] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

prints "[08:28:15] [Runner[1]] [Failed to start running. Please compile first.]" on Run command

### Expected Behavior

without compilation for Python
![Screenshot 2023-06-12 at 8 29 54](https://github.com/cpeditor/cpeditor/assets/59924334/67876d6e-4f0f-4a30-86f8-2898f71dbc35)

### Steps To Reproduce

1. Parse problem from chrome extension
2. write code
3. Run

### CP Editor Version

6.10.3

### OS

mac os m1

### Are you willing to contribute to this?

- [ ] I'm ready to fix this (and will open a PR soon)
- [ ] I'd like to have a try (with the help of the maintainers)
- [ ] No, thanks

**Comment by ouuan** [comment]
The error message is misleading for Python. It actually means that there's something wrong with your Python Run Command in Preferences.

**Reply by romankurnovskii** [reply]
nothing changed there

**Reply by ouuan** [reply, marked as answer]
Please check that you can run `python` in your terminal. Or you can try specifying the full absolute path to the Python interpreter instead of simply `python`.

**Reply by romankurnovskii** [reply]
`python` from terminal works, but setting full path in settings helps.
Thank you
